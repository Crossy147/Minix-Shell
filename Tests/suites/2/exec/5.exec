
cat /dev/null > $outf

echo "# builtin kill test 1" > $inf

$BIN/catcher >> $outf &
$BIN/catcher >> $outf &

ps | grep catcher| grep -v "grep" |  sed -n -e 's/^[ ]*\([0-9][0-9]*\).*/lkill -10 \1 /p' > $inf


$TESTED_SHELL < $inf 2> $errf >> $outf

