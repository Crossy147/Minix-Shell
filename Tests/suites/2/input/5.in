lkill -10 19619 
lkill -10 19620 
