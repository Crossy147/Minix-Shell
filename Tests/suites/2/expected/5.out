signalled by 10 
signalled by 10 
